docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -1,35 +1,28 @@
 [[detections-ui-exceptions]]
 == Rule exceptions
 
-Rule exceptions can be associated with detection and endpoint rules to prevent those rules from generating alerts under specific circumstances. Exceptions are made up of three main components: exception items, default rule lists, and shared exception lists.
+Rule exceptions can be associated with detection and endpoint rules to prevent those rules from generating alerts under specific circumstances. You can use exceptions to reduce the number of false positives, and to prevent trusted processes and network activity from generating unnecessary alerts.
```

Comment:
Is there a difference between `false positives` and `alerts under specific circumstances`? I understand these two to be the same, if not similar. Correct me if I'm wrong. Optional suggestion:

```suggestion
You can associate rule exceptions with detection and endpoint rules to prevent trusted processes and network activity from generating unnecessary alerts, therefore, reducing the number of false positives. 
```

Reply:
False positive alerts can be generated under specific circumstances so the two are definitely related. I like the revision you suggested, @jmikell821. It keeps the important notice that trusted processes and network activity can sometimes generate unnecessary alerts (i.e., false positives).